### PR TITLE
Maven project compiler plugin added

### PR DESCRIPTION
**Pull request by dexterousDangi**

### CODE REVIEW

File: target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.properties
Comment (dangi13) [inline target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.properties]: lets remove this file and add a .`gitignore` file to this repo to not include target folder and these type of files

File: target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml
Comment (dangi13) [inline target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml]: Remove this file too. only by adding target folder in .gitignore would not include this file next time.

Comment (dangi13) [reply target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml]: Lets make it a practice to use versions that will not change so frequently in properties tag , like this.
```
   <name>Flipkart Web Automation</name>
<properties>
		<maven.compiler.source>1.8</maven.compiler.source>
		<maven.compiler.target>1.8</maven.compiler.target>
</properties>
```

and then use in plugin like this

```
<groupId>org.apache.maven.plugins</groupId>
<artifactId>maven-compiler-plugin</artifactId>
<version>3.7.0</version>
<configuration>
<source>${maven.compiler.source}</source>
<target>${maven.compiler.target}</target>
</configuration>
```

**Review by dangi13 — COMMENTED** [review]
Left some comnents. Lets fix them and then after second review merge them

File: target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml
Comment (dexterousDangi) [reply target/classes/META-INF/maven/web.automation/flipkart.web.automation/pom.xml]: now check that out

**Review by dangi13 — APPROVED** [review]
Looks good. Approved.

**Pull request merged**
